app/assets/javascripts/show_hide.coffee

```diff
@@ -0,0 +1,28 @@
+root = exports ? this
+
+# Shows supplied element when radio with ID equal to 'data-show-when' value is checked.
+# Hides supplied element when any other radio in fieldset is checked.
+class ShowHide
+
+  constructor: (elementToShow) ->
+    @elementToShow = elementToShow
+    checkInputId = elementToShow.data("show-when")
+
+    @checkInput = $( "#" + checkInputId )
+    fieldset = @checkInput.closest("fieldset")
```

Comment:
Good point. I'll add checking fieldset is present in the view spec. For now the contract when using this js is that fieldset is around the radio buttons.